services/src/test-library/src/main/java/org/openkilda/testing/service/aswitch/ASwitchVirtualImpl.java

```diff
@@ -53,7 +53,7 @@ public void removeFlows(List<ASwitchFlow> flows) {
 
     @Override
     public List<ASwitchFlow> getAllFlows() {
-        throw new RuntimeException("getAllFlows operation for a-switch is not available on virtual env");
+        throw new UnsupportedOperationException("getAllFlows operation for a-switch is not available on virtual env");
```

Comment:
Just a thought - maybe it'd be better to return an empty list instead of throwing an exception? If this is not an option, could you please add "throws" clause to the method's signature?

Reply:
1. No. It will be a misleading response. This operation is not implemented for virtual env.
2. UnsupportedOperationException is unchecked exception. Adding checked exception to method signature will cause a change in the ASwitch interface and in ASwitchHardware implementation.
Won't do